docker-compose.yml

```diff
@@ -5,25 +5,27 @@ services:
     image: inputoutput/cardano-node:1.20.0
     restart: on-failure
     environment:
-      NETWORK:
+      - NETWORK=${NETWORK:-mainnet}
     volumes:
-      - node-${NETWORK}-db:/data
+      - node-db:/data
       - node-ipc:/ipc
     logging:
-      driver: none
+      driver: "json-file"
+      options:
+        max-size: "400k"
+        max-file: "20"
 
   ogmios:
     image: ktorz/ogmios:latest
     restart: on-failure
     environment:
-      - OGMIOS_NETWORK=${NETWORK}
+      - OGMIOS_NETWORK=${NETWORK:-mainnet}
     command: ["--host", "0.0.0.0", "--node-socket", "/ipc/node.socket"]
     volumes:
       - node-ipc:/ipc
     ports:
-      - 1337:1337
+      - ${OGMIOS_WS_PORT:-1337}:1337
 
 volumes:
-  node-mainnet-db:
-  node-testnet-db:
```

Comment:
I used separate volumes here originally to avoid overwriting existing volumes when switching from mainnet to testnet and vice-versa. Ideally, I'd like to write `node-{$NETWORK}-db:` but that's not (yet?) supported.

Reply:
The suggestion here is to provide a project name to set the scope at the stack level, enabling simultaneous operation of networks, and a clean boundary when it comes time to clean things up.

``` console
NETWORK=testnet docker-compose -p cardano-ogmios-testnet up
```
This way it's possible to manage the stack independently, eg

``` console
docker-compose -p  cardano-ogmios-testnet down -v
```